modules/Behaviour/behaviour_manage_edit.php

```diff
@@ -100,18 +97,42 @@
                 $form->addHiddenValue('address', "/modules/Behaviour/behaviour_manage_add.php");
                 $form->addRow()->addClass('hidden')->addHeading('Step 1', __('Step 1'));
 
+
+                //To show other students involved in the incident
+                if(!empty($values['gibbonMultiIncidentID'])) {
+
+                    $resultList = $container->get(BehaviourGateway::class)->selectMultipleStudentsOfOneIncident($values['gibbonMultiIncidentID']);
+                }
+                $students = $resultList->fetchAll();  
+            }
+
                 //Student
                 $row = $form->addRow();
                     $row->addLabel('students', __('Student'));
                     $row->addTextField('students')->setValue(Format::name('', $values['preferredNameStudent'], $values['surnameStudent'], 'Student'))->readonly();
                     $form->addHiddenValue('gibbonPersonID', $values['gibbonPersonID']);
 
+                //Other Students
+                if (!empty($values['gibbonMultiIncidentID'])) {
+                $row = $form->addRow();
+                    $row->addLabel('otherStudents0', __('Other Students Involved'));
+                    $col = $row->addColumn()->addClass('flex flex-col');
+
+                    foreach ($students as $i => $student) {
+
+                        if ($student['gibbonPersonID'] != $values['gibbonPersonID']) {
+                        $url = $session->get('absoluteURL').'/index.php?q=/modules/Students/student_view_details.php&gibbonPersonID='.$student['gibbonPersonID'].'&subpage=Behaviour&search=&allStudents=&sort=surname,preferredName';
```

Comment:
This is correct, and you'll see lots of links in the codebase like this. We are refactoring towards using the Url class, which is a PHP standard now. Can you see about implementing this link with the Url::fromModuleRoute method, eg: `$url = Url::fromModuleRoute('Behaviour', 'behaviour_manage_edit')->withQueryParams([... etc.]);`